src/Analyzer.Cli/CommandLineParser.cs

```diff
@@ -103,6 +136,64 @@ private void RootCommandHandler(FileInfo templateFilePath, FileInfo parametersFi
             }
         }
 
+        private void AnalyzeDirectory(DirectoryInfo directoryPath)
+        {
+            try {
+                // Find files to analyze
+                List<FileInfo> filesToAnalyze = new List<FileInfo>();
+                FindTemplateInDirectory(directoryPath, filesToAnalyze);
```

Comment:
Add some Console.WriteLines for how many files were identified in the directory. Maybe print it at the end. Something like "Analyzed 10 files. Unable to analyze 2." if there were a total of 12 template files, but two had errors and couldn't be analyzed. This would be especially useful when there were no template files. "Analyzed 0 files" would be a helpful message to know it couldn't find any template files.